### ISSUE TITLE
isssue with graphql . "no schema available

### ISSUE THREAD

**Issue by Gabriel-Andy** [wp-graphql/wp-graphql#1479]
hello everyone.
it is my first time opening  an issue on github.
I am building my portfolio website with gatsby.js and strapi.
I set all my contentTypes and data on on strapi. 
I open two terminals, one for strapi and one for gatsby.
to test everything, I ran the command   "npm run develop " on strapi and I have a status 200.
**this is the response https://www.screencast.com/t/YcIiZv9n**
also, I made a request by typing  http://localhost:1337/jobs and I got my data back 
this is the link that point to my response **https://www.screencast.com/t/lwFXPjY3w.**
I opened  another terminal to run "gatsby develop"
this is the response I get back **https://www.screencast.com/t/hsjVmM0CYFb**
now when I try to access  **http://localhost:8000/___graphql**
what I got returned is that "no schema available ".
this is the link that point to it **https://www.screencast.com/t/0FYKHxgn**
when I console.log to figured out what is happening
this is what I got on my console.log **https://www.screencast.com/t/Zo0MSeEnDU**

Gatsby CLI version: 2.12.101
Gatsby version: 2.24.7

"strapi": "3.1.6",
   "strapi-admin": "3.1.6",
 in summary, I can access my resource on strapi but when I request the same data using graphql like this
http://localhost:8000/___graphql nothing comes back. I basically have as returned "no schema is available
I look forward to hearing from you guys back
thanks

**Comment by jasonbahl** [comment]
Hello @Gabriel-Andy! 

This repository is for WPGraphQL, which is a WordPress plugin that provides a GraphQL API to WordPress sites. 

Unfortunately I don't have any experience with Strapi so I'm not sure I'll be able to help much here, but I'll do my best. 

Are you using [gatsby-source-strapi](https://www.gatsbyjs.com/plugins/gatsby-source-strapi/)? And have you followed the setup instructions [here](https://www.gatsbyjs.com/plugins/gatsby-source-strapi/)?

Do you have the project on a Github repository you could share? This will be the best way for someone else to try and reproduce what you're experiencing.

**Comment by Gabriel-Andy** [comment]
hi 
thank you so much for reaching back to me. 
Yes, I do use gatsby-source-strapi. and I followed all the instructions.
this is the gatsby repository https://github.com/Gabriel-Andy/my-portfolio-site
this is the strapi repo https://github.com/Gabriel-Andy/portfolio-api
 I will also share those issues on strapi repository and gatsby repository
thanks very much.
I look forward to hearing from you back

**Comment by Gabriel-Andy** [comment]
thanks so much 
someone emailed me this morning and told me to update my gatsby version to the newest one.
I am mad because I couldn't think about that.
schema for me.
